package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "migration",
+  "version": "1.0.0",
+  "description": "## Work Graph",
+  "main": "index.js",
+  "dependencies": {
+    "http-server": "^0.9.0",
+    "js-yaml": "^3.6.1",
+    "lodash": "^4.16.4",
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2"
+  },
+  "devDependencies": {
+    "neutrino": "^2.2.0",
+    "neutrino-preset-react": "^1.3.0"
+  },
+  "scripts": {
+    "start": "PORT=3000 neutrino start -p migration-preset",
```

Comment:
You can add a flag to the package.json if you want to use the same preset for all targets:

```json
{
  "config": {
    "preset": "migration-preset"
  },
  "scripts": {
    "start": "PORT=3000 neutrino start",
    "test": "neutrino test",
    "build": "neutrino build"
  }
}
```